meta.txt

```diff
@@ -1 +1 @@
6400011470726963652801304f7074696f6e3c753132383e000800284f66666572416464656410013c6e66745f636f6e74726163745f6964040128436f6e7472616374496400013866745f636f6e74726163745f696418013c4f7074696f6e3c4163746f7249643e000120746f6b656e5f69641c011c546f6b656e496400011470726963652c011075313238000900344f66666572416363657074656410013c6e66745f636f6e74726163745f6964040128436f6e74726163744964000120746f6b656e5f69641c011c546f6b656e49640001246e65775f6f776e657204011c4163746f72496400011470726963652c011075313238000a002057697468647261770c013c6e66745f636f6e74726163745f696404011c4163746f724964000120746f6b656e5f69641c011c546f6b656e496400011470726963652c011075313238000b00445472616e73616374696f6e4661696c6564000c0040526572756e5472616e73616374696f6e000d00345472616e7366657256616c7565000e00003408246d61726b65745f696f184d61726b657400001c012061646d696e5f696404011c4163746f72496400012c74726561737572795f696404011c4163746f72496400013074726561737572795f66656510010c7531360001146974656d7338019442547265654d61703c28436f6e747261637449642c20546f6b656e4964292c204974656d3e000158617070726f7665645f6e66745f636f6e74726163747370014442547265655365743c4163746f7249643e000154617070726f7665645f66745f636f6e74726163747370014442547265655365743c4163746f7249643e00011474785f69642401345472616e73616374696f6e4964000038042042547265654d617008044b013c04560140000400680000003c00000408041c004008246d61726b65745f696f104974656d00001c0120746f6b656e5f69641c011c546f6b656e49640001146f776e657204011c4163746f72496400013866745f636f6e74726163745f69641801484f7074696f6e3c436f6e747261637449643e00011470726963652801344f7074696f6e3c50726963653e00011c61756374696f6e44013c4f7074696f6e3c41756374696f6e3e0001186f66666572734c01b842547265654d61703c284f7074696f6e3c436f6e747261637449643e2c205072696365292c204163746f7249643e00010874785c01844f7074696f6e3c285472616e73616374696f6e49642c204d61726b65745478293e00004404184f7074696f6e04045401480108104e6f6e6500000010536f6d6504004800000100004808246d61726b65745f696f1c41756374696f6e00001401286269645f706572696f6424010c753634000128737461727465645f617424010c753634000120656e6465645f617424010c75363400013463757272656e745f70726963652c0114507269636500013863757272656e745f77696e6e657204011c4163746f72496400004c042042547265654d617008044b015004560104000400540000005000000408182c0054000002580058000004085004005c04184f7074696f6e04045401600108104e6f6e6500000010536f6d65040060000001000060000004082464006408246d61726b65745f696f204d61726b6574547800011c3443726561746541756374696f6e0000000c42696408011c6163636f756e7404011c4163746f72496400011470726963652c0114507269636500010034536574746c6541756374696f6e0002001053616c65040114627579657204011c4163746f724964000300144f666665720c011466745f6964040128436f6e7472616374496400011470726963652c0114507269636500011c6163636f756e7404011c4163746f7249640004002c4163636570744f666665720005002057697468647261770c011466745f6964040128436f6e7472616374496400011470726963652c0114507269636500011c6163636f756e7404011c4163746f72496400060000680000026c006c000004083c40007004204254726565536574040454010400040074000000740000020400
```

Comment:
Since `meta.txt` is put in Releases, you can add it to `.gitignore` & `git rm meta.txt`.

Reply:
Good note